node/v2/index.js

```diff
@@ -27,6 +27,7 @@ module.exports.Types = Types;
 
 var RetryFlags = require('../retry-flags.js');
 var Frame = require('./frame');
+var LazyFrame = require('./frame');
```

Comment:
Did you do a comparison with master or just eyeball it ?

Might just be a difference in machines.